pkg/inventory/inventory-client.go

```diff
@@ -268,6 +272,18 @@ func (cic *ClusterInventoryClient) getClusterInventoryInfo(localInv *unstructure
 	return clusterInv, nil
 }
 
+func (cic *ClusterInventoryClient) UpdateLabels(inv InventoryInfo, labels map[string]string) error {
```

Comment:
Should we add a unit test for this new function?

Reply:
Filed #317 for this. We'll address it in a future PR.